java/src/main/java/org/liberator/ratdriver/control/OperaControl.java

```diff
@@ -71,7 +96,23 @@ public OperaControl(OperaPreferences operaPreferences) {
      */
     @Override
     public WebDriver StartDriver() {
-        return null;
+        try {
+            System.setProperty("webdriver.opera.driver", BaseSettings.OperaDriverLocation);
+
+            SetOperaOptions();
+            SetOperaDriverService();
+
+            if (Service != null && Options != null) {
+                Driver = new OperaDriver(Service, Options);
+            } else if (Options == null && Service != null) {
+                Driver = new OperaDriver(Service);
+            } else if (Options != null) {
+                Driver = new OperaDriver(Options);
+            }
+            return Driver;
+        } catch (Exception ex) {
+            return null;
```

Comment:
why do we want to `return null` when we catch an exception?

Reply:
Ultimately, this will include an error handler class and the return will be irrelevant. The NULL is just there as a placeholder and not the final code for Maven